config/application.rb

```diff
@@ -13,6 +13,9 @@
 Bundler.require(*Rails.groups)
 
 module Skills
+  def self.use_ptime_sync?
+    ActiveModel::Type::Boolean.new.cast(ENV.fetch('USE_PTIME_SYNC', true))
```

Comment:
Does it make sense for the default to be true?
Isn't it more likely that when `USE_PTIME_SYNC is not set  all the other needed ENV variables probably aren't set either? 
I understand that this is for the local development as well, maybe we can set it somewhere? Or should it just be disabled locally :thinking: